### ISSUE TITLE
Need new PSSA code fix that injects the SuppressMessageAttribute in the right place.

### ISSUE THREAD

**Issue by rkeithhill** [PowerShell/PowerShellEditorServices#338]
Title pretty much says it all.  Although I can imagine "finding" the right place to put the attribute might be tricky.

**Comment by daviwil** [comment]
Yeah, it'd be nice to have a "suppress" recommendation that can add it.

**Comment by bergmeister** [comment]
This would become much easier once PSSA can suppress on a per line basis. There is an issue for that but no timeline but you could grab the issue.
